src/components/CivicSettingsV1.vue

```diff
@@ -215,10 +220,14 @@ export default {
   methods: {
     ...mapActions(['fetchUserSubscriptionInfo', 'resumeCanceledSubscription']),
 
-    async fetchSubscriptionInfo() {
-      if (this.getUserIsCivicLikerPaid) {
+    async fetchSubscriptionInfo({ force = false } = {}) {
+      if (this.getUserIsCivicLikerPaid || this.getUserShouldRenewCivic) {
         try {
-          const { willCancel } = await this.fetchUserSubscriptionInfo();
+          if (force || !this.getUserSubscriptionInfo) {
+            await this.fetchUserSubscriptionInfo();
+          }
+          const { willCancel, status } = this.getUserSubscriptionInfo;
+          this.isPastDueUser = status === 'past_due';
```

Comment:
turn into getter?

Reply:
Easier to unify in case later we want to do `fetchUserSubscriptionInfo()` action outside panel, maybe in the parent page

Reply:
You mean turn `isPastDueUser` into getter?

Reply:
Will open another PR